assembl/fabfile.py

```diff
@@ -2132,6 +2132,56 @@ def set_ssl_certificates():
         print(yellow("Can't set ssl certificates, env.ocsp_path is not set"))
 
 
+@task
+def create_backup_script():
+    """
+    Generates backup script that stores the backup on a local borg repository.
+    Sets a cron job for it.
+    """
+    from jinja2 import Environment, FileSystemLoader
+    print(green("Generating backup script on remote host."))
+    jenv = Environment(loader=FileSystemLoader('doc/'), autoescape=lambda t: False)
+    backup_template = jenv.get_template('backup_template.jinja2')
```

Comment:
Once again, you can use the fill_template option.

Reply:
not sure I understand what you mean @ayyazdaniaryan

Reply:
never mind, I just saw the nginx pull request. I understand now what you mean 👍